src/configure/configure.ts

```diff
@@ -427,11 +442,13 @@ class Orchestrator {
     }
 
     private async getSelectedPipeline(): Promise<void> {
-        var repoAnalysisHelper = new RepoAnalysisHelper(this.inputs.azureSession);
-        var repoAnalysisResult = await repoAnalysisHelper.getRepositoryAnalysis(this.inputs.sourceRepository,
-            this.inputs.pipelineConfiguration.workingDirectory.split('/').join('\\'));
+        const repoAnalysisHelper = new RepoAnalysisHelper(this.inputs.azureSession, this.inputs.githubPATToken);
+        let repoAnalysisResult = null;
+        if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
+            repoAnalysisResult = await repoAnalysisHelper.getRepositoryAnalysis(this.inputs.sourceRepository, this.inputs.pipelineConfiguration.workingDirectory.split('\\').join('/'));
```

Comment:
Are we making sure that we rely on `repoAnalysisResult` only in case of github repo.

Reply:
Yes that's what this check is for